src/App.tsx

```diff
@@ -323,12 +506,15 @@ class Board extends React.Component<BoardProps, BoardState> {
                         <div className="button-box-bot">
                             <EventPreventingButton
                                 variant="contained"
-                                color="primary">
+                                color="primary"
+                                onClick={() => {this.restart(); }}>
```

Comment:
Perhaps you can consider enabling/disabling the button if undo or redo is possible